docs/content/kubernetes-tutorial.md

```diff
@@ -25,13 +25,18 @@ For the purpose of the tutorial we will deploy two policies that ensure:
 
 ## Prerequisites
 
-This tutorial requires Kubernetes 1.13 or later. To run the tutorial locally ensure you start a cluster with Kubernetes version 1.13+, we recommend using [minikube](https://kubernetes.io/docs/getting-started-guides/minikube) or [KIND](https://kind.sigs.k8s.io/).
+This tutorial requires Kubernetes 1.13 or later. To run the tutorial locally ensure you start a cluster with Kubernetes
+version 1.13+, we recommend using [minikube](https://kubernetes.io/docs/getting-started-guides/minikube) or
+[KIND](https://kind.sigs.k8s.io/).
 
 ## Steps
 
-### 1. Start Kubernetes recommended Admisson Controllers enabled
+### 1. Start Kubernetes recommended Admission Controllers enabled
```

Comment:
👏 😄

Reply:
"Start Kubernetes recommended Admission Controllers enabled" still reads a bit weird to me though?
Should it not just be "Enable recommended Kubernetes Admission Controllers"?

Reply:
Updated the step text.